lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -27,6 +27,8 @@ use DBIx::Class::Timestamps 'now';
 use Mojo::Asset::Memory;
 use Mojo::File 'path';
 
+use constant JOB_QUERY_LIMIT => 10000;
```

Comment:
why 10k? According to the error in the ticket the limit that we *have* internally is 64k. I don't see why we should limit this further.

Reply:
Because 10000 jobs already take quite a while to process on my fast machine which had nothing else to do at this time. Not only the query limit is a constraint here. In the end this outputs quite a bunch of information about *each* job (including the job settings). Serving that amount of data for 64k jobs is insane. Likely the connection times out before that would happen anyways. And even more likely the user aborts the client before that will happen.